src/app/home/home.component.ts

```diff
@@ -35,32 +36,32 @@ export class HomeComponent implements OnInit {
 
   private highlightCards: readonly HighlightCard[] = [
     {
-      appearAt: 'asap',
-      disappearAt: CCOpenDate,
-      headerText: 'Christmas Challenge 2023 is starting soon!',
-      bodyText: 'Save the date 28th November - 5th December 2023',
-      iconColor: "brand-cc-red",
-      backgroundImageUrl: new URL('/assets/images/card-background-cc-lights.jpg', environment.donateGlobalUriPrefix),
+      appearAt: Jan8th,
+      disappearAt: Feb9th,
```

Comment:
I assumed this and the one below have to disappear on Feb9th - is that right @BethBigGive and if so do we want to replace them with anything?